app/(tabs)/wallet.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
-import { View, StyleSheet, ScrollView, TouchableOpacity } from 'react-native';
+import { View, StyleSheet, ScrollView, TouchableOpacity, Image } from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 import { ThemedText } from '@/components/ThemedText';
 import { ThemedView } from '@/components/ThemedView';
 import { Link } from 'expo-router';
 import { Colors } from '@/constants/Colors';
+import Typography from '@/components/Typography/Typography';
```

Comment:
Add index file to remove `Typography/Typography` import 

```suggestion
import Typography from '@/components/Typography';
```